docs/dev/setup/jenkins-localvc.rst

```diff
@@ -0,0 +1,797 @@
+        url: http://172.17.0.1:8080 # `http://host.docker.internal:8080` for Windows
+
+In addition, you have to start Artemis with the profiles ``localvc`` and
+``jenkins`` so that the correct adapters will be used, e.g.:
+
+::
+
+   --spring.profiles.active=dev,jenkins,localvc,artemis,scheduling
+
+Please read :ref:`Server Setup` for more details.
+
+For a local setup on Windows you can use `http://host.docker.internal` appended
+by the chosen ports as the continuous-integration url and the internal vcs url.
+
+Make sure to change the ``server.url`` and ``artemis.version-control.url`` value in ``application-dev.yml``
+or ``application-prod.yml`` accordingly. The ``server.url`` value will be used for the
+communication hooks from Jenkins to Artemis.
+In case you use a different port than 80 (http) or 443 (https) for the
+communication, you have to append it to the both urls value,
+e.g. \ ``127.0.0.1:8080``.
+
+When you start Artemis for the first time, it will automatically create
+an admin user.
+
+**Note:** Sometimes Artemis does not generate the admin user which may lead to a startup
+error. You will have to create the user manually in the MySQL database. Make sure
+both are set up correctly and follow these steps:
+
+1.  Use the tool mentioned above to generate a password hash.
+2.  Connect to the database via a client like `MySQL Workbench <https://dev.mysql.com/downloads/workbench/>`__
+    and execute the following query to create the user. Replace `artemis_admin` and `HASHED_PASSWORD` with your
+    chosen username and password:
+
+    .. code:: sql
+
+        INSERT INTO `artemis`.`jhi_user` (`id`,`login`,`password_hash`,`first_name`,`last_name`,`email`,
+        `activated`,`lang_key`,`activation_key`,`reset_key`,`created_by`,`created_date`,`reset_date`,
+        `last_modified_by`,`last_modified_date`,`image_url`,`last_notification_read`,`registration_number`)
+        VALUES (1,"artemis_admin","HASHED_PASSWORD","artemis","administrator","artemis_admin@localhost",
+        1,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL);
+3. Give the user admin and user roles:
+
+    .. code:: sql
+
+        INSERT INTO `artemis`.`jhi_user_authority` (`user_id`, `authority_name`) VALUES (1,"ROLE_ADMIN");
+        INSERT INTO `artemis`.`jhi_user_authority` (`user_id`, `authority_name`) VALUES (1,"ROLE_USER");
+
+Starting the Artemis server should now succeed.
+
+Jenkins
+^^^^^^^
+
+Automated Jenkins Server Setup
+""""""""""""""""""""""""""""""
+
+The following steps describe how to deploy a pre-configured version of the Jenkins server.
+This is ideal as a quickstart for developers. For a more detailed setup, see
+`Manual Jenkins Server Setup <#manual-jenkins-server-setup>`__.
+In a production setup, you have to at least change the user credentials (in the file ``jenkins-casc-config-localvc.yml``) and push tokens.
+
+1. You can now first build and deploy Jenkins, then you can also start the other services which weren't started yet:
```

Comment:
```suggestion
#. You can now first build and deploy Jenkins, then you can also start the other services which weren't started yet:
```
Same for other three items below to automatically get correct numbering. At the moment there are only steps 1, 2, 5, 6.